Destructuring/generic-functions.lisp

```diff
@@ -44,6 +44,9 @@
 ;;; Return LET* bindings for a list of &KEY parameters.
 (defgeneric key-parameters-bindings (client parameters argument-variable))
 
+;;; Return LET* bindings for validating a &KEY parameter group.
+(defgeneric key-validation-bindings (client parameter-group argument-variable))
```

Comment:
Maybe document second return value?

Reply:
Since all the functions in the file return these same values, I think the file comment near the top displaying them is better for this. I tried rephrasing it to make it clearer.

Reply:
I didn't look closely enough to spot that, sorry.